src/index.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/store';
 import App from './App';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <BrowserRouter>
-    <App />
+    <Provider store={store}>
+      <App />
+    </Provider>
   </BrowserRouter>,
 );
```

Comment:
- [ ] Good job setting up your poriject but when I'm running `npm start ` on my PC i'm not able to see the results , the project is bringing the error below , I could be on my side but make sure that your project setting are proper too 👍 

![Capture](https://user-images.githubusercontent.com/80458199/229766947-a82a12fb-3739-42d0-bc48-655a9003c22b.PNG)